tests/integration/utils.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+include './src/lib/kwio.sh'
+
+declare -g CONTAINER_DIR # has the container files to build the container images
+declare -g SAMPLES_DIR   # has sample files used accross the integration tests
+declare -g KWROOT_DIR    # local kw dir to be copied to and installed in the containers
+declare -g DISTROS       # distributions we will run the integration tests
+
+# ensure path to directories is absolute
+script_dir=$(realpath "$(dirname "${0}")")
+CONTAINER_DIR="${script_dir}/podman"
+SAMPLES_DIR="${script_dir}/samples"
+KWROOT_DIR=$(realpath "${script_dir}/../..")
+
+# supported distros
+DISTROS=('archlinux' 'debian' 'fedora')
+
+# Builds a container imagem for the given distro
+function build_distro_image()
+{
+  local distro="${1}"
+  local file="${CONTAINER_DIR}/Containerfile_${distro}"
+
+  podman image build --file "$file" --tag "kw-${distro}" 2> /dev/null
```

Comment:
I suppose this command may fail, right? If so, you must handle the return (`$?`) value.